src/components/AccordionSection/check-is-in-array.js

```diff
@@ -0,0 +1,3 @@
+export default function isInArray(array, item) {
```

Comment:
Let's change the file name to be the same the function name: is-in-array. If we would use this name then should be check-if-is-in-array.